packages/multi-select-combo-box/src/vaadin-multi-select-combo-box.js

```diff
@@ -510,6 +510,15 @@ class MultiSelectComboBox extends ResizeMixin(InputControlMixin(ThemableMixin(El
     return this.required && !this.readonly ? this._hasValue : true;
   }
 
+  /**
+   * Clear the selected items.
+   */
+  clear() {
```

Comment:
I would suggest to also redeclare this method in the Typescript defs. Currently when navigating to the definition of `comboBox.clear` in an IDE it navigates to `input-mixin.d.ts` which documents the method as "Clear the value of the field.".

Reply:
Good point, done.